static/styles/dark_theme.css

```diff
@@ -278,6 +278,14 @@
         }
     }
 
+    textarea.new_message_textarea {
+        &.invalid,
+        &.invalid:focus {
+            border-color: 3, 73%, 74%;
```

Comment:
This was invalid CSS -- you meant `hsl(3, 73%, 74%)`, Fixing that typo makes it look like this, which is a lot better contrast:

![image](https://user-images.githubusercontent.com/2746074/220800290-c1f34019-4240-4b09-bd17-f83b3b98d3ca.png)

I'm not sure it's the ideal color but seems ok to integrate this.